app-server/src/main/scala/rf/healthcheck/Healthcheck.scala

```diff
@@ -0,0 +1,55 @@
+package com.azavea.rf.healthcheck
+
+import scala.concurrent.ExecutionContext
+
+import com.azavea.rf.utils.Database
+
+
+/**
+  * Individual service check for a component (database, cache, etc.)
+  * 
+  * @param service name of service that check is for
+  * @param status status of service (e.g. OK, UNHEALTHY)
+  */
+case class ServiceCheck(service: String, status: String)
```

Comment:
Might be good to use case classes or similar for the service/status to have a clear set of options defined and so they don't need to be passed around as strings.